source/modules/MEI.cmn.xml

```diff
@@ -875,17 +875,16 @@
     </attList>
   </classSpec>
   <classSpec ident="att.numberPlacement" module="MEI.cmn" type="atts">
-    <desc>Attributes that record the placement and visibility of numbers that accompany a bowed
-      tremolo or tuplet.</desc>
+    <desc>Attributes that record the placement and visibility of numbers that accompany a notational object.</desc>
     <attList>
       <attDef ident="num.place" usage="opt">
-        <desc>States where the tuplet number will be placed in relation to the note heads.</desc>
+        <desc>Indicates where the number is written in relation to its reference object.</desc>
```

Comment:
I think this is a correct modification reflecting MEI's transcriptional perspective

Reply:
as @ahankinson minted out the undefinedness of `reference object` produces a problem here since data.STAFFREL.basic itself defines the staff as reference object.